create_derivatives/routines.py

```diff
@@ -128,8 +132,121 @@ def ocr_pdf(self):
+            self.fac.set_base_prezi_dir(manifest_dir)
+            self.create_manifest(jp2_files, manifest_dir, str(Path(bag.bag_path, "data", "JP2")), bag.dimes_identifier, bag.as_data)
+            bag.process_status = Bag.MANIFESTS_CREATED
+            bag.save()
+            bags_with_manifests.append(bag.dimes_identifier)
+        msg = "Manifests successfully created." if len(bags_with_manifests) else "No manifests created."
+        return msg, bags_with_manifests
+
+    def create_manifest(self, files, manifest_dir, image_dir, identifier,
+                        obj_data):
+        """Method that runs the other methods to build a manifest file and populate
+        it with information.
+
+        Args:
+            files (list): Files to iterate over
+            manifest_dir (str): Path to directory to write manifests to.
+            image_dir (str): Path to directory containing derivative image files.
+            identifier (str): A unique identifier.
+            obj_data (dict): Data about the archival object.
+        """
+        manifest_path = "{}.json".format(str(Path(manifest_dir, identifier)))
+        page_number = 1
+        manifest = self.fac.manifest(ident="{}{}".format(self.fac.prezi_base, identifier), label=obj_data["title"])
+        manifest.set_metadata({"Date": obj_data["dates"]})
+        manifest.thumbnail = self.set_thumbnail(Path(files[0]).stem)
+        sequence = manifest.sequence(ident=identifier)
+        for file in files:
+            page_ref = Path(file).stem
+            width, height = self.get_image_info(image_dir, file)
+            canvas = sequence.canvas(
+                ident="{}/canvas/{}".format(
+                    manifest.id, str("{0:03}".format(page_number))),
+                label="Page {}".format(
+                    str(page_number)))
+            canvas.set_hw(height, width)
+            annotation = canvas.annotation(ident=page_ref)
+            img = annotation.image(
+                ident="/{}/full/max/0/default.jpg".format(page_ref))
+            self.set_image_data(img, height, width, page_ref)
+            canvas.thumbnail = self.set_thumbnail(page_ref)
+            page_number += 1
+        v2_json = manifest.toJSON(top=True)
+        v3_json = self.upgrader.process_resource(v2_json, top=True)
+        with open(manifest_path, 'w', encoding='utf-8') as jf:
+            json.dump(v3_json, jf, ensure_ascii=False, indent=4)
+
+    def get_image_info(self, image_dir, file):
+        """Gets information about the image file.
+
+        Args:
+            image_dir (str): path to the directory containing the image file
+            file (str): filename of the image file
+        Returns:
+            width (int): Pixel width of the image file
+            height (int): Pixel height of the image file
+        """
+        with Image.open(str(Path(image_dir, file))) as img:
+            width, height = img.size
+        return width, height
```

Comment:
Can you just return directly here?
```suggestion
            return img.size
```